utils/automation/fossologyscanner.py

```diff
@@ -82,13 +83,14 @@ def get_allow_list() -> dict:
   return data
 
 
-def print_results(name: str, failed_results: List[ScanResult],
+def print_results(name: str, failed_results: List[ScanResult],scan_results_with_line_number:List[dict],
```

Comment:
Please keep the line length limited to 80 characters :-). Please put the new parameter on new line.

Reply:
Okay! I'll keep that in mind!